independent-publisher-connectors/Shopify Admin Store/apiProperties.json

```diff
@@ -0,0 +1,151 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2generic",
+          "clientId": "",
```

Comment:
@thatapiguy Please provide the value as `"[[Provide your client id]]"` or `"[[TEST]]"` or "[[DUMMY]]"`